templates/sec-template.txt

```diff
@@ -46,34 +53,54 @@ notification) to these data nodes. Specifically, the following
 subtrees and data nodes have particular sensitivities/
 vulnerabilities:
 
-<list subtrees and data nodes and explain the reasons for
- the sensitivity/privacy concerns>
-
- -- if your YANG module has defined any RPC operations
- -- describe their specific sensitivity or vulnerability.
+-- If the data model contains any particularly sensitive readable
+-- data nodes, e.g., ones that might be protected by a
+-- "nacm:default-deny-read" or a "nacm:default-deny-all" extensions
+-- statement, or if they may reveal sensitive customer information
+-- or violate personal privacy laws, such as those of the Euopean
+-- Union, if exposed to unauthorized parties, then those subtrees
+-- and data nodes must be listed here, along with an explanation of
+-- the associated sensitivity, security, or privacy concerns.
+--
+-- Otherwise, state:
+-- "There are no particularly sensitive readable data nodes."
+
+-- RPC operations section:
+--
+-- If the data model contains any RPC operations, then include
+-- the following section of text:
 
 Some of the RPC operations in this YANG module may be considered
 sensitive or vulnerable in some network environments.  It is thus
 important to control access to these operations.  Specifically,
 the following operations have particular sensitivities/
 vulnerabilities:
 
-<list RPC operations and explain the reasons for the sensitivity/
- privacy concerns>
-
-   -- if your YANG module reuses groupings from other modules and
-   -- the document that specifies these groupings also
-   -- includes those as data nodes, then add this text to remind
-   -- the specific sensitivity or vulnerability of reused nodes.
+-- If the data model contains any particularly sensitive RPC
+-- operations, then those operations must be listed here, along
+-- with an explanation of the associated sensitivity or privacy
+-- concerns.
+--
+-- Otherwise, state:
+-- "There are no particularly sensitive RPC operations."
+
+-- Reusable groupings from other modules section:
+-- 
+-- If the data model reuses groupings from other modules and
+-- the document that specifies these groupings also
+-- includes those as data nodes, then add this text to remind
+-- the specific sensitivity or vulnerability of reused nodes.
 
 This YANG module uses groupings from other YANG modules that
 define nodes that may be considered sensitive or vulnerable
 in network environments. Refer to the Security Considerations
 of <RFC-insert-numbers> for information as to which nodes may
 be considered sensitive or vulnerable in network environments.
 
-  -- if your YANG module does not define any data nodes, then
-  -- add the following text
+-- No data nodes section:
+-- 
+-- If the data model does not define any data nodes, then
```

Comment:
Thanks, Rob. 

Fixed at: https://github.com/netmod-wg/rfc8407bis/commit/0d3697c077d67143fff5c2149f469c475c27337c. 

One minor edit, went with "action" instead of "Action" to be consistent with RFC7950.